docs/performance/benchmark.md

```diff
@@ -0,0 +1,23 @@
+# Benchmark
+
+This document explain how to perform a benchmark of approximate nearest neighbor on the Vald cluster, such as [ann-benchmarks](https://github.com/erikbern/ann-benchmarks).
```

Comment:
I think this document will show not only ann-benchmarks, cuz code bench sometimes will not relate to ann-benchmark but only performance (transport, load balance, filtering, vectorize, etc...)